parsec/core/fs/remote_loader.py

```diff
@@ -325,13 +343,26 @@ def __init__(
                 expected_author=expected_author,
                 expected_timestamp=expected_timestamp,
                 expected_version=expected_version,
-                expected_id=entry_id
-                # TODO: check parent as well ?
+                expected_id=entry_id,
             )
         except DataError as exc:
             raise FSError(f"Cannot decrypt vlob: {exc}") from exc
 
-        # TODO: also store access id in remote_manifest and check it here
+        # Finally make sure author was allowed to create this manifest
+        role_at_timestamp = await self._get_user_realm_role_at(
+            expected_author.user_id, expected_timestamp
+        )
+        if role_at_timestamp is None:
+            raise FSError(
+                f"Manifest was created at {expected_timestamp} by `{expected_author}` "
+                "which had no right to access the workspace at that time"
+            )
+        elif role_at_timestamp == RealmRole.READER:
+            raise FSError(
+                f"Manifest was created at {expected_timestamp} by `{expected_author}` "
+                "which had write right on the workspace at that time"
```

Comment:
`which had no write right`? `which had no right to write in the workspace`?

Reply:
wrong wording or poor style ?